### PR TITLE
RTF-1288 - Add examples for ILibraryApplicationService

### PR DESCRIPTION
**Pull request by jonatanjakoniuk**
Complete the following tasks before merging in your PR:
- [x] If this is a user facing change, update [version.txt](https://github.com/relativitydev/relativity.testing.framework.api/blob/master/version.txt) and [CHANGELOG.md](https://github.com/relativitydev/relativity.testing.framework.api/blob/master/CHANGELOG.md)

### CODE REVIEW

**Review by paulletang — APPROVED** [review]

File: source/Relativity.Testing.Framework.Api/Services/ILibraryApplicationService.cs [hunk 1]
```diff
@@ -33,13 +47,26 @@ public interface ILibraryApplicationService
...
 		/// Gets the library application by the specified name.
 		/// </summary>
 		/// <param name="name">The name.</param>
 		/// <returns>The <see cref="LibraryApplication"/> entity or <see langword="null"/>.</returns>
+		/// <example>
+		/// <code>
+		/// string applicationName = "CuttingEdgeAppName";
+		/// LibraryApplication application = _libraryApplicationService.Get(applicationName);
+		/// </code>
+		/// </example>
```
Comment (briantoms) [inline source/Relativity.Testing.Framework.Api/Services/ILibraryApplicationService.cs:69]: Feels weird that we have a get by name but no get by ID. Not related to the changes in this PR, though.

Comment (paulletang) [reply source/Relativity.Testing.Framework.Api/Services/ILibraryApplicationService.cs:69]: Based on the new working agreements, should we make a story to address this gap?

Comment (briantoms) [reply source/Relativity.Testing.Framework.Api/Services/ILibraryApplicationService.cs:69]: I think we should make a story if the underlying Relativity APIs expose get by ID. If not, there's not much we can do.

Comment (paulletang) [reply source/Relativity.Testing.Framework.Api/Services/ILibraryApplicationService.cs:69]: Looks like it doesn't exist.
https://platform.relativity.com/RelativityOne/Content/BD_Environment/Library_Application_service.htm#_Read_a_library_application

We would need to get all the applications, and then filter by name.

Comment (briantoms) [reply source/Relativity.Testing.Framework.Api/Services/ILibraryApplicationService.cs:69]: It's literally the first one there. `<host>/relativity.rest/api/relativity-environment/{versionNumber}/workspace/-1/libraryapplications/{artifactID}`

We could also think about adding GetAll at the same time.

Comment (briantoms) [reply source/Relativity.Testing.Framework.Api/Services/ILibraryApplicationService.cs:69]: Get by name is what we have in RTF today, but that doesn't seem to be supported by the V1 API.

**Review by matthew-wiemer — APPROVED** [review]

File: source/Relativity.Testing.Framework.Api/Services/ILibraryApplicationService.cs (hunk 1 above)
Comment (paulletang) [reply source/Relativity.Testing.Framework.Api/Services/ILibraryApplicationService.cs:69]: Sorry, I switched them in my head. 😕
Let me see if we already have a story for this.

**Review by klaudiaknafel — APPROVED** [review]

**Pull request merged**